### DISCUSSION TITLE
Loki: Add new type of template variable - query_result

### DISCUSSION THREAD

**Discussion by loganmc10** [grafana/grafana#37805] [Data Sources]
I'd like to do this: https://www.robustperception.io/graph-top-n-time-series-in-grafana using Loki. This will allow me to graph top N series using Loki metric/unwrapped data.

Doing this requires doing a topk query and getting the label values back, which requires the query_result() function

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
